packages/dds/tree/src/feature-libraries/default-schema/schemaChecker.ts

```diff
@@ -29,6 +29,9 @@ export const enum SchemaValidationErrors {
 	UnknownError,
 }
 
+/**
+ * Deeply checks that the provided node complies with the schema based on its identifier.
```

Comment:
Previous use of this called it at every level in the tree. Since this is a deep recursive checker this was incorrect usage and it now has a doc comment making that clear.